### PR TITLE
Hydraulic hdf

### PR DESCRIPTION
**Pull request by codymccoy**
The name of this branch is not named well. This has nothing to do with hydro hdf. This branch was swapping out the CA file and folder browser for the new ones.

### CODE REVIEW

File: UserControls/packages.config [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="HEC.MVVMFramework.Base" version="1.0.0-alpha-2022.7.8.2331" targetFramework="net48" />
+  <package id="HEC.MVVMFramework.Base" version="1.0.0-alpha-2022.7.20.1519" targetFramework="net48" />
   <package id="HEC.MVVMFramework.Model" version="1.0.0-alpha-2022.7.8.2331" targetFramework="net48" />
   <package id="RASMapperLibDistro6.2" version="0.0.1" targetFramework="net48" />
   <package id="System.Collections.Immutable" version="1.5.0" targetFramework="net48" />
-  <package id="USACE.HEC.FDA.Model" version="1.1.0-alpha-2022-07-19-1451" targetFramework="net48" />
```
Comment (Brennan1994) [inline UserControls/packages.config:7]: I'm confused how this keeps coming back, but please delete all these package references from Model, RASMapperDistro, Framework Model and Base, and statistics. Super weird. I feel like you've deleted these before.

File: View/Study/CreateNew.xaml.cs [hunk 2]
```diff
@@ -12,11 +12,11 @@ public CreateNew()
             InitializeComponent();
         }
 
-        private void TextBoxFolderBrowser_SelectionMade(string fullpath)
```
Comment (Brennan1994) [inline View/Study/CreateNew.xaml.cs:15]: can we just delete rather than comment out code.

File: View/Study/ImportStudyFromFDA1.xaml.cs [hunk 3]
```diff
@@ -15,13 +15,13 @@ public ImportStudyFromFDA1()
             InitializeComponent();        
         }
 
-        private void TextBoxFolderBrowser_SelectionMade(string fullpath)
```
Comment (Brennan1994) [inline View/Study/ImportStudyFromFDA1.xaml.cs:18]: just delete unused code

File: View/WaterSurfaceElevation/WaterSurfaceElevationImporter.xaml.cs [hunk 4]
```diff
@@ -15,14 +15,14 @@ public WaterSurfaceElevationImporter()
             InitializeComponent();
         }
 
-        private void TxtDirectory_SelectionMade(string fullpath)
```
Comment (Brennan1994) [inline View/WaterSurfaceElevation/WaterSurfaceElevationImporter.xaml.cs:18]: delete commented code

**Review by Brennan1994 — CHANGES_REQUESTED** [review]

File: UserControls/packages.config (hunk 1 above)
Comment (codymccoy) [reply UserControls/packages.config:7]: I will try a wooden stake and a silver bullet this time.

**Pull request merged**